lib/urlwatch/jobs.py

```diff
@@ -198,6 +199,13 @@ def retrieve(self, job_state):
         if result != 0:
             raise ShellError(result)
 
+    bytes_filters = [name for name, class_ in FilterBase.__subclasses__.items()
+                     if getattr(class_, '__uses_bytes__', False)]
```

Comment:
Now that is in two places, maybe we can make this a classmethod on `FilterBase`?

    @classmethod
    def is_bytes_filter(cls, filter):
        return (filter in [name for name, class_ in FilterBase.__subclasses__.items()
                           if getattr(class_, '__uses_bytes__', False)])

Reply:
Oh, I am learning lots from you.  On to it.